precompile/deposit/src/lib.rs

```diff
@@ -17,7 +17,6 @@
 // along with Darwinia. If not, see <https://www.gnu.org/licenses/>.
 
 #![cfg_attr(not(feature = "std"), no_std)]
-#![deny(unused_crate_dependencies)]
```

Comment:
Why remove this rule for these crates?

Reply:
While compiling to WASM, it complains about `unused crate std, alloc`.